lib/gitspindle/github.py

```diff
@@ -1502,15 +1505,17 @@ def whois(self, opts):
                 continue
             emails = {}
             if user.login == self.my_login:
-                for email in self.gh.iter_emails():
-                    emails[email['email']] = email
+                for email in self.gh.emails():
+                    emails[str(email)] = email
```

Comment:
`str(email)` looks off, should't we be using an attribute?

Reply:
done